python/hsfs/statistics.py

```diff
@@ -36,56 +51,156 @@ def __init__(
         type=None,
     ):
```

Comment:
Type hinting return type
```suggestion
    ) -> "Statistics":